### DISCUSSION TITLE
GitHub Action cache size

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [JetBrains/Qodana#128] [General]
Qodana takes 1.3GB cache (x3, so almost 4GB) in a repository with a total cache of 10GB.

Is it possible to only cache the main branch and the rest of the branches should use that cache (similar behavior to Gradle GitHub Action)?

**Comment by tiulpin** [comment]
Hi! Thank you for the idea, it sounds like a feature worth implementing!

I created an issue for that https://youtrack.jetbrains.com/issue/QD-4283, the updates will be published there
